app/controllers/api/v1/skills_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::SkillsController < ApplicationController
+
+  def create
+    begin
+      user = User.find(params[:user_id])
+
+      params[:skills].each do |skill|
+        user.skills.create(name: skill[:name], proficiency: skill[:proficiency] )
+        #could we have other error handling here for skills already being present for user? 
```

Comment:
Hmm yeah that would be an interesting method. I wonder if we could do something like checking the user's skills beforehand in a helper method and if the added skill name is already included in the array of the user's skills then it would send a message that the skill already exists? It's something that can be added later if we wanted